saleor/tests/e2e/account/test_should_create_account_without_email_confirmation.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+
+from ..channel.utils import create_channel
+from ..shop.utils import update_shop_settings
+from ..utils import assign_permissions
+from .utils import account_register, token_create
+
+
+@pytest.mark.e2e
+def test_should_create_account_without_email_confirmation_core_1502(
+    e2e_not_logged_api_client,
+    e2e_staff_api_client,
+    permission_manage_channels,
+    permission_manage_settings,
+):
+    # Before
+    permissions = [permission_manage_channels, permission_manage_settings]
+
+    assign_permissions(e2e_staff_api_client, permissions)
```

Comment:
I think we could skip the gap between those lines. It's one operation of assigning permissions to users.